README.md

```diff
@@ -1 +1,3 @@
-# TCR_sequence_analysis
\ No newline at end of file
+# TCR sequence analysis
+
```

Comment:
This seems like a great idea but pedantic

Reply:
Do you have a suggestion of something I should to improve it?